### PR TITLE
AIMS-297 Add more info to item detail page

### PR DESCRIPTION
**Pull request by rfox2**
What: Aaron requested that we indicate when an item was dissociated from a tray on the item detail page, which I did and also added several other transitions to the list.
How:  Added several actions to the item_history method and cleaned up the activity_log_query.rb file a bit.

### CODE REVIEW

**Comment by jaronkk** [comment]
:+1:

File: app/services/activity_logger.rb [hunk 1]
```diff
@@ -159,7 +159,7 @@ def log!
       data: data,
       username: username,
       user_id: user_id,
-      action_timestamp: Time.now,
+      action_timestamp: Time.zone.now,
```
Comment (jgondron) [inline app/services/activity_logger.rb]: Will rails convert this to UTC when writing to the db?

Comment (jaronkk) [inline app/services/activity_logger.rb]: I think Rails should have already been converting this to UTC in the database, so I don't know if this change makes any difference.  @rfox2 ?

Comment (rfox2) [inline app/services/activity_logger.rb]: You're right. I'll remove it.

**Comment by jgondron** [comment]
:+1:

**Pull request closed without merging**